### PR TITLE
Elixir AAI Integration

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/hca-ebi-dev-team#48

### CODE REVIEW

File: client/src/app/aai/aai.service.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  getUser(): Promise<User> {
+    return this.manager.getUser();
+  }
+
+  isLoggedIn(): Observable<boolean> {
+    return from(this.manager.getUser()).map(user => user && !this.user.expired);
+  }
+
+  isAuthenticated(): boolean {
+    return this.user && !this.user.expired;
```
Comment (rolando-ebi) [inline client/src/app/aai/aai.service.ts]: Some of these methods can throw null exceptions if the user hasn't yet been fetched from the UserManager. Could fix this by wrapping access to `this.user` in `getUser()` only, which returns a Observable<User> from manager.getUser() if `this.user` is null, or returns an `Observable.of(this.user)` it's not null. Might have to rename the existing `getUser()` then to something like fetch-user

Comment (aaclan-ebi) [reply client/src/app/aai/aai.service.ts]: Applied comments.

**Comment by aaclan-ebi** [comment]
Will wait for the Ingest Core API changes before merging.

**Comment by aaclan-ebi** [comment]
merging now, deploying to dev

**Pull request merged**